src/ripple/rpc/tests/LedgerClosed.test.cpp

```diff
@@ -0,0 +1,64 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2016 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <ripple/protocol/JsonFields.h>
+#include <ripple/protocol/Feature.h>
+#include <ripple/test/jtx.h>
+
+namespace ripple {
+
+class LedgerClosed_test : public beast::unit_test::suite
+{
+public:
+
+    void testMonitorRoot()
+    {
+        using namespace test::jtx;
+        Env env {*this};
+        Account const alice {"alice"};
+        env.fund(XRP(10000), alice);
+
+        auto lc_result = env.rpc("ledger_closed") [jss::result];
+        BEAST_EXPECT(lc_result[jss::ledger_hash]  == "8AEDBB96643962F1D40F01E25632ABB3C56C9F04B0231EE4B18248B90173D189");
+        BEAST_EXPECT(lc_result[jss::ledger_index] == 2);
+
+        env.close();
+        auto const ar_master = env.le(env.master);
+        BEAST_EXPECT(ar_master->getAccountID(sfAccount) == env.master.id());
```

Comment:
This test of master account id seems redundant as it is testing jtx internals only, If I understand correctly